build/add_head.mjs

```diff
@@ -0,0 +1,35 @@
+import fs from "node:fs";
+import path from "node:path";
+
+const favicon = `
+<link rel="apple-touch-icon" href="/docs/images/favicons/apple-touch-icon.png" sizes="180x180">
+<link rel="icon" href="/docs/images/favicons/favicon-32x32.png" sizes="32x32" type="image/png">
+<link rel="icon" href="/docs/images/favicons/favicon-16x16.png" sizes="16x16" type="image/png">
+<link rel="manifest" href="/docs/images/favicons/manifest.json">
+<link rel="mask-icon" href="/docs/images/favicons/safari-pinned-tab.svg" color="#000">
+<link rel="icon" href="/docs/images/favicons/favicon.ico">
+<meta name="msapplication-config" content="/docs/images/favicons/browserconfig.xml">
+<meta name="theme-color" content="#000">`;
+const boosted_CSS = `
+<link href="https://cdn.jsdelivr.net/npm/boosted@5.3.3/dist/css/orange-helvetica.min.css" rel="stylesheet" integrity="sha384-A0Qk1uKfS1i83/YuU13i2nx5pk79PkIfNFOVzTcjCMPGKIDj9Lqx9lJmV7cdBVQZ" crossorigin="anonymous">
+<link href="https://cdn.jsdelivr.net/npm/boosted@5.3.3/dist/css/boosted.min.css" rel="stylesheet" integrity="sha384-laZ3JUZ5Ln2YqhfBvadDpNyBo7w5qmWaRnnXuRwNhJeTEFuSdGbzl4ZGHAEnTozR" crossorigin="anonymous">
+<link href="/docs/assets/tarteaucitron-config.css" rel="stylesheet">`;
```

Comment:
I got an issue with the Netlify deployed version for the `/api` URL:

<img width="1461" alt="Screenshot 2024-03-29 at 10 26 17" src="https://github.com/Orange-OpenSource/ods-charts/assets/17381666/32fbb301-0f77-4c82-bb19-7c3d6cad67ba">